README.md

```diff
@@ -1,5 +1,306 @@
+
+## Basic Definitions
+
+This project controls two major resource types on Azure Media Services: Streaming Endpoints and Live Events. Here are their definitions:
+
+- **Live Events:** A channel that a broadcaster will reuse to publish similar events over and over again.
+- **Streaming Endpoint:** The ingest mechanism on Azure Media Services which allows a broadcaster to connect their streaming software of choice, such as [Wirecast](https://www.telestream.net/wirecast/) or [OBS Studio](https://obsproject.com/), and begin sending a live stream.
+
+## Installation
+
+As mentioned before, this application is intended to run on Azure Functions. Therefore, the instructions are geared toward the process of installing the API there. Overall, there are several significant phases to the installation:
+
+1. [Setting up the Media Services resources on Azure](#Setting-up-the-Media-Services-Resources-on-Azure)
+1. [Creating the Azure Functions application](#Creating-the-Azure-Functions-Application)
+1. [Setting up the environment variables](#Setting-up-the-Environment-Variables)
+1. [Deploying the application](#Deploying-the-Application)
+1. [Authenticating with the application](#Authenticating-with-the-Application)
+1. [Set up the streaming software](#Set-up-the-Streaming-Software)
+
+Guides for using the API as a [broadcaster](https://github.com/literal-life-church/live-streaming-controller) and [viewer](https://github.com/literal-life-church/stream-switch) are available in the READMEs of their respective repositories.
+
+### Setting up the Media Services Resources on Azure
+
+Obviously, to use this application, you'll need an active Azure account. Please [sign up for an account](https://signup.azure.com/) if you don't already have one. Once you have an account set up, [sign in to the Azure Portal](https://portal.azure.com/).
+
+#### Create the Media Service
+
+First, begin by creating the Media Service resource.
+
+1. From the search bar at the top, search for "Media Services," and then select the "Media Services" service
+1. Click the Add button to create a new Media Service
+1. Enter an Account Name, choose your active Subscription, and a Location
+1. For the resource group, create a new one
+1. Click on Storage Account &gt; Create New
+1. Enter a Storage Account Name, and leave all other settings as the default
+1. Back on the Media Service creation screen, click the Create button
```

Comment:
Codacy found an issue: [[ordered-list-marker-value] Marker should be `7`, was `1`](https://app.codacy.com/gh/literal-life-church/live-streaming-api/pullRequest?prid=5728997)